### DISCUSSION TITLE
PR RAT

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#303] [General]
Author: @kathypinzon 
Reviewer: @AbbyANoble 

https://www.overleaf.com/read/xppqbrsjqgkx#3bb84b

**Comment by kathypinzon** [comment]
@AbbyANoble @tdegeorge  this one needs a reviewer.... any takers?

**Reply by AbbyANoble** [reply]
I'll take it since I reviewed your readiness.

**Comment by AbbyANoble** [comment]
Answers not listed in the answer key.

**Comment by AbbyANoble** [comment]
Question 3 - Extra parenthesis at the end of each answer choice.

**Comment by AbbyANoble** [comment]
Question 7 - Not a complete sentence. I'd change it to "Let f(x)=stuff. Find f(-4)." 

And add \dfrac to the function.

Answer should be (b). Or change the denominator to x-3 to get choice (c).

**Comment by AbbyANoble** [comment]
Question 8 - "On which interval is the graph decreasing?"  Not intervals (plural) since all the answer choices are just one interval.

**Reply by AbbyANoble** [reply]
And the answer should be (a) there, not (b).

**Comment by AbbyANoble** [comment]
Question 9 - answer should be (c) not (d)

**Comment by AbbyANoble** [comment]
Question 10 - 

(a) has x-intercepts as plural but only one listed
(b) and (d) need a comma after the second x-intercept

**Comment by AbbyANoble** [comment]
Question 11 - I don't think we really need this. They will probably just use a calculator which defeats the point. I think it makes more sense for it to be an activity in that section like we talked about before.

**Comment by AbbyANoble** [comment]
I'm not seeing why Find the equation of a line given two points is in the readiness? Maybe I'm missing something, but where do we use that in this chapter?

**Comment by AbbyANoble** [comment]
Desmos graphs instead of tikz graphs make these require multiple files if we upload to github. It's not that big of a deal since we will probably be the only ones using these in the fall, but I thought I'd point it out. Would need changing for easier sharing to others probably.
